lib/msal-core/src/utils/WindowUtils.ts

```diff
@@ -107,6 +107,23 @@ export class WindowUtils {
         });
     }
 
+    /**
+     * @hidden
+     * Loads the iframe synchronously when the navigateTimeFrame is set to `0`
+     * @param urlNavigate
+     * @param frameName
+     * @param logger
+     */
+    static syncLoadFrame(urlNavigate: string, frameName: string, logger: Logger): HTMLIFrameElement{
```

Comment:
Could this be `loadFrameSync`, to keep the convention consistent?

Reply:
👍